.mypy/emptybaseline.json

```diff
@@ -0,0 +1,2 @@
+{
+}
```

Comment:
whys this needed

Reply:
because there's no `--no-baseline` flag.

Reply:
i can run basedmypy without a baseline file just fine. it does so in the basedtyping project

Reply:
I want to ignore the baseline in this invokation, and it will git an error if the file doesn't exist.

Reply:
#95